osu.Game/Scoring/ScoreInfo.cs

```diff
@@ -66,6 +66,11 @@ public class ScoreInfo : RealmObject, IHasGuidPrimaryKey, IHasRealmFiles, ISoftD
         [MapTo("MaximumStatistics")]
         public string MaximumStatisticsJson { get; set; } = string.Empty;
 
+        /// <summary>
+        /// Hash of the beatmap where it scored
+        /// </summary>
+        public string OriginalBeatmapHash { get; set; } = string.Empty;
```

Comment:
This doesn't make sense chronologically (shouldn't be "original" because it's not like it will be updated or anything).

```suggestion
        /// <summary>
        /// The beatmap's <see cref="BeatmapInfo.Hash"> at the point in time when the score was set.
        /// </summary>
        public string BeatmapHash { get; set; } = string.Empty;
```

Reply:
```csharp
/// <summary>
        /// The beatmap's <see cref="BeatmapInfo.Hash"/> at the point in time when the score was set.
        /// </summary>
        public string BeatmapHash { get; set; } = string.Empty;
```
idk why but this one fails code quality test